docs/antrea-ipam.md

```diff
@@ -218,6 +218,34 @@ metadata:
 A StatefulSet Pod's IP will be kept after Pod restarts, when the IP is allocated from the
 annotated IPPool.
 
+### IPAM behaviors
+
+When `AntreaIPAM` is enabled, `antrea-controller` will watch IPPool CRs and StatefulSets
+from K8s apiserver, and `antrea-agent` will register IPAM driver to CNI server.
+
+#### On IPPool CR create/update event
+
+antrea-controller will update IPPool usage.
+
+#### On StatefulSet create event
+
+antrea-controller will check annotations and will preallocate IPs from IPPool.
+
+#### On StatefulSet delete event
+
+antrea-controller will clean IP allocations under this StatefulSet.
+
+#### On Pod create
+
+antrea-agent will receive CNI add request and will check annotations and allocate IP from
+IPPool. The IP is a preallocated StatefulSet IP, a specified IP or a next available IP in
+this IPPool.
```

Comment:
```suggestion
`antrea-agent` will receive a CNI add request, it will check annotations and allocate an IP from
an IPPool. The IP is a pre-allocated StatefulSet IP, a specified IP or the next available IP in
the IPPool.
```